packages/pieces/community/mollie/src/lib/actions/create-payment-link.ts

```diff
@@ -0,0 +1,59 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { mollieAuth } from '../common/common';
+import { MollieApi } from '../common/common';
+
+export const createPaymentLinkAction = createAction({
+  auth: mollieAuth,
+  name: 'create_payment_link',
+  displayName: 'Create Payment Link',
+  description: 'Generate a new payment link targeting a customer, product, or specific amount',
+  props: {
+    amount: Property.Number({
```

Comment:
Amount must be optional and a decimal string

```suggestion
    amount_currency: Property.StaticDropdown({
  displayName: 'Amount Currency',
  required: false,
  defaultValue: 'EUR',
  options: { options: [{ label: 'EUR', value: 'EUR' }, { label: 'USD', value: 'USD' }, { label: 'GBP', value: 'GBP' }] },
}),
amount_value: Property.ShortText({
  displayName: 'Amount Value',
  description: 'Exact amount string with 2 decimals (e.g. 10.00)',
  required: false,
}),
minimumAmount_currency: Property.StaticDropdown({
  displayName: 'Minimum Amount Currency',
  required: false,
  defaultValue: 'EUR',
  options: { options: [{ label: 'EUR', value: 'EUR' }, { label: 'USD', value: 'USD' }, { label: 'GBP', value: 'GBP' }] },
}),
minimumAmount_value: Property.ShortText({
  displayName: 'Minimum Amount Value',
  description: 'Exact amount string with 2 decimals (e.g. 1.00). Only when no amount is set.',
  required: false,
}),
```